### PR TITLE
Supports getting column name from Row

### PR DESCRIPTION
**Pull request by pratikpparikh**
This is an enhancement for #135

### CODE REVIEW

**Comment by EmadAlblueshi** [comment]
Hi @pratikpparikh, 

I think we need test for `Row` in [RowTest](https://github.com/reactiverse/reactive-pg-client/blob/master/src/test/java/io/reactiverse/pgclient/RowTest.java)

File: src/main/java/io/reactiverse/pgclient/impl/RowImpl.java [hunk 1]
```diff
@@ -45,6 +46,12 @@ public RowImpl(RowImpl row) {
     this.desc = row.desc;
   }
 
+  @Override
+  public String getColumnName(int pos) {
+    final ColumnDesc[] columnDescs = desc.columns();
+    return pos == -1 || columnDescs.length < pos ? null : columnDescs[pos].getName();
```
Comment (EmadAlblueshi) [inline src/main/java/io/reactiverse/pgclient/impl/RowImpl.java]: I think if pos < 0 would be better to make sure of all negative numbers

Comment (pratikpparikh) [reply src/main/java/io/reactiverse/pgclient/impl/RowImpl.java]: i am adding a negative test as well.

**Comment by pratikpparikh** [comment]
@EmadAlblueshi we should be covered not with test cases and appropriate code check-in for this request.

File: src/test/java/io/reactiverse/pgclient/RowTest.java [hunk 2]
```diff
@@ -97,6 +97,44 @@ public void testGetNonExistingRows(TestContext ctx) {
...
+        ctx.asyncAssertSuccess(result -> {
+          Row row = result.iterator().next();
+          ctx.assertEquals("foo",row.getColumnName(0));
+          async.complete();
+        }));
+    }));
+  }
+
+  @Test
+  public void testNegativeNotEqualGetColumnNameRows(TestContext ctx) {
```
Comment (EmadAlblueshi) [inline src/test/java/io/reactiverse/pgclient/RowTest.java]: Should rename the test method without **negative**

File: src/test/java/io/reactiverse/pgclient/RowTest.java [hunk 3]
```diff
@@ -97,6 +97,44 @@ public void testGetNonExistingRows(TestContext ctx) {
...
+  }
+
+  @Test
+  public void testNegativeGetColumnNameRows(TestContext ctx) {
+    Async async = ctx.async();
+    PgClient.connect(vertx, options, ctx.asyncAssertSuccess(conn -> {
+      conn.query("SELECT 2 \"foo\"",
+        ctx.asyncAssertSuccess(result -> {
+          Row row = result.iterator().next();
+          ctx.assertEquals(null,row.getColumnName(1));
```
Comment (EmadAlblueshi) [inline src/test/java/io/reactiverse/pgclient/RowTest.java]: Should use `ctx.assertNull(row.getColumnName(-1))` instead

Comment (pratikpparikh) [reply src/test/java/io/reactiverse/pgclient/RowTest.java]: @EmadAlblueshi this is still a valid test case (edge length test case), maybe we need to add an additional test case.

File: src/test/java/io/reactiverse/pgclient/RowTest.java (hunk 2 above)
Comment (pratikpparikh) [reply src/test/java/io/reactiverse/pgclient/RowTest.java]: @EmadAlblueshi okay.

File: src/test/java/io/reactiverse/pgclient/RowTest.java (hunk 3 above)
Comment (EmadAlblueshi) [reply src/test/java/io/reactiverse/pgclient/RowTest.java]: @pratikpparikh Yes because this test should reflects its name :)

Comment (pratikpparikh) [reply src/test/java/io/reactiverse/pgclient/RowTest.java]: @EmadAlblueshi agreed.  The test cases are now named appropriately.

Comment (pratikpparikh) [reply src/test/java/io/reactiverse/pgclient/RowTest.java]: @EmadAlblueshi we should be good with test case naming now as well.

**Comment by vietj** [comment]
I'll merge it

**Comment by vietj** [comment]
thanks for the contribution!

**Pull request merged**